pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4391,6 +4368,24 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		Expect(resp.Result.Details.Causes).To(HaveLen(1))
 		Expect(resp.Result.Message).To(ContainSubstring("no memory requested"))
 	})
+
+	It("should allow Clock without Timer", func() {
+		vmi := newBaseVmi(withDomainClock(
```

Comment:
Could you please elaborate why libvmi isn't directly used?
Same for the other tests.

Reply:
It serves as a wrapper for specifying memory, without it tests would fail due to: https://github.com/kubevirt/kubevirt/blob/37b59b2e0f55ae845d6fe6966dfec24092acd9b1/pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go#L1194-L1213